src/components/article-page/article-page.jsx

```diff
@@ -19,15 +21,26 @@ function ArticlePage() {
         console.log(err);
       }
     }
+    async function fetchCommentsByArticleID() {
+      try {
+        setLoadingComments(true);
+        const commentsFromAPI = await getCommentsByArticleID(article_id);
+        setCommentsArray(commentsFromAPI);
+        setLoadingComments(false);
+      } catch (err) {
+        console.log(err);
+      }
+    }
```

Comment:
I'd declare the function outside of the useEffect and only call it within the useEffect